src/containers/collectibles/components/CollectiblesPage.tsx

```diff
@@ -0,0 +1,629 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import {
+  Button,
+  ButtonSize,
+  ButtonType,
+  IconLink,
+  IconPencil,
+  Modal
+} from '@audius/stems'
+import cn from 'classnames'
+import styles from './CollectiblesPage.module.css'
+import PerspectiveCard from 'components/perspective-card/PerspectiveCard'
+import DynamicImage from 'components/dynamic-image/DynamicImage'
+import UserBadges from 'containers/user-badges/UserBadges'
+import { DragDropContext, Draggable, Droppable } from 'react-beautiful-dnd'
+import { ReactComponent as IconVolume } from 'assets/img/iconVolume.svg'
+import { ReactComponent as IconMute } from 'assets/img/iconVolume0.svg'
+import { ReactComponent as IconPlay } from 'assets/img/pbIconPlay.svg'
+import { ReactComponent as IconShow } from 'assets/img/iconMultiselectAdd.svg'
+import { ReactComponent as IconHide } from 'assets/img/iconRemoveTrack.svg'
+import { ReactComponent as IconDrag } from 'assets/img/iconDrag.svg'
+import { ReactComponent as IconGradientCollectibles } from 'assets/img/iconGradientCollectibles.svg'
+import Tooltip from 'components/tooltip/Tooltip'
+import { Collectible, CollectiblesMetadata, CollectibleType } from './types'
+import { ProfileUser } from 'containers/profile-page/store/types'
+import { formatDate } from 'utils/timeUtil'
+import Drawer from 'components/drawer/Drawer'
+import Spin from 'antd/lib/spin'
+
+const VISIBLE_COLLECTIBLES_DROPPABLE_ID = 'visible-collectibles-droppable'
+
+const messages = {
+  title: 'COLLECTIBLES',
+  subtitlePrefix: 'A collection of NFT collectibles owned and created by '
+}
+
+// @ts-ignore
+const VisibleCollectible = props => {
```

Comment:
Would try to type unless there's something really hard about adding types for this 🙃

Reply:
there was something annoying about it, then I didn't circle back. good catch